### ISSUE TITLE
Circle morphs into square

### ISSUE THREAD

**Issue by peller** [maqetta/maqetta#3789]
Drag a circle on the page
In the properties panel for borders, select the border value and either delete it or select "remove value"
Circle changes into square

**Comment by JonFerraiolo** [comment]
When you say "changes into square", do you mean that the selection chrome is rectangular? If you remove the border, the circle is invisible because it has no background/fill and it has no border, our selection chrome is always around the (x,y) bounds of the widget.

However, if you put a background on the circle, and then remove the border, yes, indeed, it does change into a square as this issue described. Not sure what to do about that - whether to document this as a "feature" or do some special widget-specific magic in the code (probably make it so that no border gets overridden to have border:1px solid transparent or something like that)

**Comment by peller** [comment]
right, the visible widget effectively changes to a square and does not mask the area outside the circle.  background colors fill the entire (square) div.  Something we could probably document our way around.
